notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -819,7 +742,7 @@
       "source": [
         "### Create VPC peering\n",
         "\n",
-        "For simplicity, we setup VPC peering to the ucaip-haystack-vpc-network network. You can create a different network for your project.\n",
+        "For simplicity, you setup VPC peering to the `ucaip-haystack-vpc-network` network. You can create a different network for your project.\n",
```

Comment:
again, don't need "you" here. Also "setup" is a noun. Should be "set up", which is a verb.
- "For simplicity, set up VPC peering to the `ucaip-haystack-vpc-network` network."